primitives/linux-port/column.cpp

```diff
@@ -62,6 +63,55 @@ inline uint64_t order_swap(uint64_t x)
   return ret;
 }
 
+// Dummy template
+template<typename T,
+        typename std::enable_if<sizeof(T) >= sizeof(uint128_t), T>::type* = nullptr>
+inline T orderSwap(T x)
+{
+    return x;
+}
+
+template<typename T,
+        typename std::enable_if<sizeof(T) == sizeof(int64_t), T>::type* = nullptr>
+inline T orderSwap(T x)
+{
+    T ret = (x >> 56) |
```

Comment:
Maybe it's better to use `bswap_*` or even `htobe*`?

Reply:
Any obvious reasons to use them ?

Reply:
Moreover there will be no 128 bit version if we later need it.

Reply:
I would like such a thing but imagine how complex it might be to support. IMHO this is an overkill.